packages/slice-machine/tests/project/cypress/integration/onboarding/00-redirects.spec.js

```diff
@@ -14,21 +14,18 @@ describe('onboarding redirects and local storage', () => {
     cy.saveLocalStorage();
   });
 
-  it('should redirect to /onboarding when is-onboared is not in local storage', () => {
+  it('should redirect to /onboarding when is-onboarded is not in local storage', () => {
     cy.intercept('POST', '/tracking/onboarding', ({body}) => {
       expect(body.id).equal("slicemachine_onboarding_start")
     })
     
     cy.visit('/')
 
     cy.location('pathname', {timeout: 1000}).should('eq', '/onboarding')
-
-    cy.wait(1500)
-    cy.getLocalStorage("is-onboarded").should('eq', 'true')
   })
 
   it('when is-onboarded is in local storage it should not redirect', () => {
-    cy.setLocalStorage("is-onboarded", true)
+    cy.setLocalStorage("persist:root", JSON.stringify({userContext: JSON.stringify({isOnboarded: true})}))
```

Comment:
isn't it a little weird to have to stringify twice ? Shouldn't it be a big object ?